routes/lang/en-application.json

```diff
@@ -14,7 +14,13 @@
   "app.addusertitle": "Add new user",
   "app.disableuser": "Disable user",
   "app.deleteuser": "Delete user",
-  "app.confirmdeleteuser": "Deleting<br/><b>%{email}</b>.<br/><br/>This is a one-way trip. Continue?",
+  "app.deleteusertransfer": "Transfer ownership to me",
+  "app.deleteuserdelete": "Delete all unshared courses",
+  "app.deleteusershare": "Share all courses",
+  "app.confirmdeleteuser": "Deleting<br/><b>%{email}</b>.<br/><br/>%{courseOption}<br/><br/>This is a one-way trip. Continue?",
+  "app.confirmdeleteusertransfer": "<b>You will be the owner of all * their unshared courses.</b>",
+  "app.confirmdeleteuserdelete": "<b>All unshared courses will be deleted.</b>",
```

Comment:
Perhaps "All of their unshared courses will be deleted." to be consistent with the other messages